### PR TITLE
ContextMenu / Menu: Apply New Data set in Context Menu and Menu component in both MVC and .Net Demos

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: MVCDemos/Models/SampleData/ContextMenuData.cs [hunk 1]
```diff
@@ -1,1243 +1,360 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using static System.Net.Mime.MediaTypeNames;
```
Comment (EugeniyKiyashko) [inline MVCDemos/Models/SampleData/ContextMenuData.cs]: Why it requires here?

Comment (nikkithelegendarypokemonster) [reply MVCDemos/Models/SampleData/ContextMenuData.cs]: [4a3cf45](https://github.com/DevExpress/devextreme-demos/pull/3211/commits/4a3cf450c084b9e548b9cab85debcc7d1b60e319)
tested this one we can safely remove the unecessary imports even the one on top of it the Linq

File: MVCDemos/Models/SampleData/ContextMenuData.cs [hunk 2]
```diff
@@ -1,1243 +1,358 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
 
 namespace DevExtreme.MVC.Demos.Models.SampleData {
     public static class ContextMenuData {
         public static readonly IEnumerable<ContextMenuScrollingItem> Products = new[] {
             new ContextMenuScrollingItem {
-                text = "Audio",
+                text = "Electronics",
```
Comment (EugeniyKiyashko) [inline MVCDemos/Models/SampleData/ContextMenuData.cs:9]: The dataSource for both demos is identical. We might be able to use only one file, but let's leave it as it is. I don't see any plans to refactor this.

**Review by EugeniyKiyashko — DISMISSED** [review]

File: MVCDemos/Models/SampleData/ContextMenuData.cs (hunk 2 above)
Comment (EugeniyKiyashko) [reply MVCDemos/Models/SampleData/ContextMenuData.cs:9]: You can resolve my comment

**Review by EugeniyKiyashko — DISMISSED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by iBat — APPROVED** [review]

**Pull request merged**